src/ray/raylet/agent_manager.cc

```diff
@@ -55,27 +55,34 @@ void AgentManager::StartAgent() {
     RAY_LOG(INFO) << "Not starting agent, the agent command is empty.";
     return;
   }
```

Comment:
I moved the stanza below to after adding the new arguments so it prints out everything